### DISCUSSION TITLE
Are the operations "hat" and "vee" (from Lie theory context) somewhere implemented in SymPy?

### DISCUSSION THREAD

**Discussion by evbernardes** [sympy/sympy#24548] [Q&A]
Basically the title.

I would like to know if there are any implementations of these operations.

More generally, the "vee" $(\cdot)^\vee$ operator takes an element of a Lie algebra to a vector of size equal to the dimension of the Lie algebra, while the "hat" $\widehat{(\cdot)}$ operator does the inverse.

What I'm actually interested in is the operations in SO(3), so that "hat" is equivalent to getting the matrix representing the cross product:
```math
\pmb a \times \pmb b = \widehat{\pmb a} \, \pmb b
```

**Comment by sylee957** [comment]
I haven't seen feature about that yet. You can browse `liealgebra` or `physics` to see if there's some feature like that silently implemented.
